code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -292,10 +292,11 @@
 		access = get_all_centcom_access()
 		..()
 
-/obj/item/weapon/card/id/centcom/ERT
+/obj/item/weapon/card/id/ERT
 	name = "\improper Emergency Response Team ID"
+	icon_state = "centcom"
 	assignment = "Emergency Response Team"
+	New()

```

Comment:
Bad proc format. Always use full proc names, unless the predominant style in that file dictates otherwise.

`New()` to `/obj/item/weapon/card/id/ERT/New()`, with the tabbing fixed.